shell/platform/tizen/tizen_window_ecore_wl2.h

```diff
@@ -34,6 +34,8 @@ class TizenWindowEcoreWl2 : public TizenWindow {
 
   void* GetRenderTarget() override { return ecore_wl2_egl_window_; }
 
+  void* GetRenderTargetContainer() override { return ecore_wl2_window_; }
```

Comment:
`TizenWindow` has a method for the same purpose. Please consider removing the existing method.
```
virtual void* GetWindowHandle();
```

Reply:
Thank you for telling me. I deleted the method that mentioned and modified the part of using this method.